mimic/model/images.py

```diff
@@ -19,6 +20,13 @@ def set_is_default(self):
         """
         self.is_default = True
 
+    @classmethod
+    def image_id(cls):
```

Comment:
I like this being random since it closer models how the api actually behaves(id's frequently changing)